### DISCUSSION TITLE
swagger

### DISCUSSION THREAD

**Discussion by bellaabdelouahab** [ISIL-ESTE/Student-Workflow-Organizer#48] [Q&A] [answered]
in swagger do you have to add the end point to docs each time you add it in router

@muttaqin1

**Comment by ghost** [comment, marked as answer]
Yes.
@bellaabdelouahab

**Reply by bellaabdelouahab** [reply]
that is not how it should be done, in my recent project with spring boot, the swagger docs generated automatically, i will see if i can fix it 
please create an issue for that

**Reply by ghost** [reply]
No need to create issues because we are not working with swagger anymore. Khalid proposed to use postman docs.
#30 
@bellaabdelouahab

**Reply by ghost** [reply]
But if you still want to use swagger i will surely create a issue. 
And i believe this will help you : https://medium.com/swlh/automatic-api-documentation-in-node-js-using-swagger-dd1ab3c78284
@bellaabdelouahab
